### PR TITLE
Add codelab for pub/sub to CF

### PR DESCRIPTION
**Pull request by fifthposition**
@dorbin @lwander Last PR for a while (I think). Apologies for constant pinging. :)

(Also added CF to list of providers on the main Setup / Install / Providers page.)

### CODE REVIEW

**Comment by fifthposition** [comment]
Ping :D

**Comment by dorbin** [comment]
@okundzich
Another one for you to take a look at?

**Comment by okundzich** [comment]
Looks good. @dorbin.

File: guides/tutorials/codelabs/pubsub-to-cf/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Now create a pull subscription, giving a subscription name (`GCP_SUB_NAME`) and the topic name used in the last command (`TOPIC`):
+
+```
+$ GCP_SUB_NAME=<Insert Subscription Name>
+$ gcloud beta pubsub subscriptions create $GCP_SUB_NAME --topic $TOPIC
+```
+
+## 3. Create a GCP Service Account
+
+Following the GCP documentation on Creating and Managing Service Accounts, create a new service account, giving it the Storage Admin (`roles/storage.admin`) and Pub/Sub Subscriber (`roles/pubsub.subscriber`) roles.
```
Comment (dorbin) [inline guides/tutorials/codelabs/pubsub-to-cf/index.md]: @fifthposition can you link to that doc from here?

**Review by dorbin — CHANGES_REQUESTED** [review]
@fifthposition 
I like how you number the headings, a good idea when the whole article is an overarching procedures, but it's not short.

Inisde each numbered-heading section, you might number the (sub-)procedure. I think if you start them all with `a` they'll auto number a., b., c. But preview it to be sure, if you're set up  to do so.

File: guides/tutorials/codelabs/pubsub-to-cf/index.md (hunk 1 above)
Comment (fifthposition) [reply guides/tutorials/codelabs/pubsub-to-cf/index.md]: Ooh. Good catch. Thanks!

**Comment by fifthposition** [comment]
Thanks for review @dorbin. Made changes!

**Comment by fifthposition** [comment]
@dorbin LMK how it looks now. I can't have it auto-number subsections but I tried doing it by hand.

**Review by dorbin — APPROVED** [review]
👍 
Thanks, @fifthposition

**Pull request merged**